server/controllers/usersController.js

```diff
@@ -115,28 +124,35 @@ export default class Users {
     })
       .then((foundUser) => {
         if (!foundUser) {
-          return res.status(404).send({
-            message: 'This email does not exist. Sign up instead ?',
-          });
+          return res.status(404)
+            .json({
```

Comment:
### Noteworthy
Your JSON responses do not comply with the [jsonapi.org specification](http://jsonapi.org).
```
Success response
{
  data: [] | {}
}

Failure response
{
  error: {
    status: 4xx | 5xx (error codes basically),
    code: Application specific error code (optional),
    title: error title,
    detail: error msg explaining why request failed
  }
}
```
With this format, I only check for one of two things: data or error